src/models/wasteWater/story/WasteWaterStoryPage.tsx

```diff
@@ -15,29 +15,136 @@ export const WasteWaterStoryPage = () => {
+        proportions of variants are less reliable in this situation.
       </p>
       <WasteWaterSamplingSites />
+      <DataSources />
+      <VideoPresentation />
+      <Acknowledgements />
+      <Funding />
+      <Contact />
+    </div>
+  );
+};
+
+const DataSources = () => {
+  return (
+    <div>
+      <h2>Data sources</h2>
+      <p>
+        <ExternalLink url={'https://www.eawag.ch/en/department/sww/projects/sars-cov2-in-wastewater/'}>
+          Eawag
+        </ExternalLink>{' '}
+        collects samples daily at six Swiss wastewater treatment plants: Altenrhein (SG), Chur (GR), Genève
+        (GE), Laupen (BE), Lugano (TI). For the City of Zurich, samples are collected from the Werdhölzli
+        plant (catchment area covers the city center). Since 2023 the project has been joined by Microsynth,
+        sampling in Lausanne (Vidy), Sierre/Noes (VS) and Porrentruy (JU). The Health Department of
+        Basel-Stadt provides samples from the ProRheno AG wastewater treatment plant three times per week
+        (once per week until the end of 2022).
+      </p>
+      <p>Previous monitoring that was discontinued: Lausanne (VD), Kanton Zürich</p>
+      <p>
+        Until end of May 2022 three times a week the Cantonal Laboratory Zurich (KLZH) provided samples that
+        assess the Canton of Zurich by pooling samples from 12 plants across the canton, namely
+        Zürich-Werdhölzli (also used by Eawag), Winterthur-Hard, Dietikon-Limmattal, Dübendorf-Neugut,
+        Niederglatt-Fischbach, Uster, Bülach-Furt, Wetzikon-Flos, Horgen-Oberrieden, Meilen, Affoltern
+        a.A.-Zwillikon, and Illnau-Mannenberg.
+      </p>
+    </div>
+  );
+};
+
+const VideoPresentation = () => {
+  return (
+    <div>
+      <h2>Video presentation of the surveillance project</h2>
+      <p>
+        During the webinar{' '}
+        <ExternalLink url={'https://iwa-network.org/learn/detecting-covid-19-variants-in-wastewater/'}>
+          "Detecting COVID-19 Variants in Wastewater" by the International Water Association (IWA)
+        </ExternalLink>
+        , Prof. Tamar Kohn and Prof. Niko Beerenwinkel have presented this surveillance project. The
+        presentation is the second session of this webinar,
+        <ExternalLink url={'https://vimeo.com/560055953#t=1337s'}>
+          "Detection and surveillance of SARS-CoV-2 genomic variants in Swiss wastewater" (timecode 22:17).
+        </ExternalLink>
+        .
+      </p>
+    </div>
+  );
+};
+
+const Acknowledgements = () => {
+  return (
+    <div>
+      <h2>Acknowledgements</h2>
+      <h3>Computational Biology Group (CBG) ETH Zürich / Swiss Institute of Bioinformatics</h3>
+      <Authors
+        authors={
+          'Katharina Jahn, Pelin Burcak Icer, David Dreifuss, Ivan Topolsky, Lara Fuhrmann, Kim Philipp Jablonski, Anika John, Niko Beerenwinkel'
+        }
+      />
+      <h3>Computational Evolution (cEvo) ETH Zürich / Swiss Institute of Bioinformatics</h3>
```

Comment:
"Computational Evolution (cEvo) ETH Zürich" should point to https://bsse.ethz.ch/cevo

"Swiss Institute of Bioinformatics" should point to https://www.sib.swiss/tanja-stadler-group